jbpm-db-scripts/src/main/resources/db/upgrade-scripts/h2/rhpam-7.13-to-7.13.1.sql

```diff
@@ -1,4 +1,4 @@
 ALTER TABLE TimerMappingInfo ADD COLUMN processInstanceId bigint;
 ALTER TABLE TimerMappingInfo ALTER COLUMN timerId bigint;
 DROP INDEX IDX_EventTypes_element;
-CREATE INDEX IDX_EventTypes_IdElement ON EventTypes(InstanceId,element);
\ No newline at end of file
+CREATE INDEX IDX_EventTypes_IdElement ON EventTypes(InstanceId,element);
```

Comment:
what is this line changing? a whitespace?

Reply:
same as previous